sunpy/net/helioviewer.py

```diff
@@ -189,6 +192,57 @@ def download_jp2(self, date, observatory=None, instrument=None, detector=None,
 
         return self._get_file(params, directory=directory, overwrite=overwrite)
 
+    def get_jp2_header(self, date, observatory=None, instrument=None, detector=None,
+                       measurement=None, jp2_id=None):
```

Comment:
Can this be made one line?

Reply:
We cannot remove anything as these are essential parameters. We can keep it in one line but it will be a too much in a single line

Reply:
Is it too long if its one line?

Reply:
I guess so.